Description: Don't build mpir or mpfr; use Debian packages instead.
 We actually use gmp instead of mpir.
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2015-06-03

--- a/M2/configure.ac
+++ b/M2/configure.ac
@@ -804,15 +804,13 @@
 BUILD_lrslib=no
 BUILD_cddplus=no
 
-BUILD_mpir=yes
-BUILD_ALWAYS="mpir $BUILD_ALWAYS"
+BUILD_mpir=no
 # We always build mpir so we can patch the routine mp_set_memory_functions() in it so
 # it does nothing; that prevents libraries such as pari from telling it what memory
 # allocation functions to use.  But we should instead patch pari, so we can use the
 # gmp library that comes with many linux distributions.
 
-BUILD_mpfr=yes
-BUILD_ALWAYS="mpfr $BUILD_ALWAYS"
+BUILD_mpfr=no
 # We always build mpfr because we tell mpir (which is impersonating gmp) to use libgc for its
 # memory allocation.  But mpfr puts pointers to gmp numbers in thread local variables, unless
 # specially configured, which libgc can't find, so libgc regards them as garbage and ultimately
